src/patterns/start-pages/index.md.njk

```diff
@@ -0,0 +1,7 @@
+---
+title: Start pages
+layout: layout-archived.njk
+ignore_in_sitemap: true
+---
+
+In March 2022, we replaced the 'Start pages' pattern with a new pattern to show teams more ways they can help users [Start using a service](/start-using-a-service/).
```

Comment:
I think this is why the broken link checker was failing, there is no page at `/start-using-a-service/`, the link needs to be either `./start-using-a-service/` or `/patterns/start-using-a-service/`.